config/config.exs

```diff
@@ -14,7 +14,7 @@ config :logger,
 
 case Mix.env() do
   :test ->
-    System.put_env("KADEMLIA_PORT", "51053")
+    System.put_env("KADEMLIA_PORT", "51_053")
```

Comment:
This is in an environment variable and would also not be understood. Needs to stay without _